src/senaite/sync/browser/views.py

```diff
@@ -427,6 +452,14 @@ def get_authenticated_user(self):
         """
         return self.get_first_item("users/current")
 
+    def get_senaite_registry_records(self):
+        """Return the values of the registry records
+        associated to bika or senaite in the source instance
+
+        :returns:
+        """
+        return self.get_json("registry/senaite-config-registry")
```

Comment:
Here you could do e.g. `self.get_json("registry/bika")` and `self.get_json("registry/senaite")` when you changed the `senaite.jsonapi` accordingly